example-config.yaml

```
@@ -6,15 +6,15 @@ homeserver:
   domain: example.com
 
   # Is the homeserver actually mautrix-asmux?
-  asmux: false
+  software: standard
   # The URL to push real-time bridge status to.
   # If set, the bridge will make POST requests to this URL whenever a user's slack connection state changes.
   # The bridge will use the appservice as_token to authorize requests.
   status_endpoint: null
   # Endpoint for reporting per-message status.
   message_send_checkpoint_endpoint: null
   # Does the homeserver support https://github.com/matrix-org/matrix-spec-proposals/pull/2246?
-  async_media: false
+  async_media: true
```

This can't be enabled by default

```suggestion
  async_media: false
```